src/test/it/bom.yml

```diff
@@ -0,0 +1,56 @@
+metadata:
+  # labels and annotations are key: value string attributes
+  labels:
+    name: bom-it
+    # Needed to make PCT happy
+    groupId: org.jenkins-ci.main
+    artifactId: jenkins-war
+    version: 2.118-artifact-manager-s3-SNAPSHOT
```

Comment:
Why do we need to specify a `version`, anyway? Could the packager not just pick something arbitrary (`999-SNAPSHOT`), or generate a version based on the `spec.core.version`?

Reply:
Current BOM specification seems to require this field. https://github.com/jenkins-infra/pipeline-library/pull/57 actually runs without it, but it will set `1.0-SNAPSHOT` as a default now. Makes sense to switch to `999-SNAPSHOT` as you propose